sound/soc/intel/boards/sof_sdw.c

```diff
@@ -704,11 +704,14 @@ static inline int get_next_be_id(struct snd_soc_dai_link *links,
 	return links[be_id - 1].id + 1;
 }
 
+#define IDISP_CODEC_MASK	0x4
```

Comment:
@kv2019i  One question: is our HDMI codec fixed at 0x4 ? thanks!

Reply:
@RanderWang Good question. This is basicly defining a bit in HDA.STATESTS register which maps to SDI interface. On all hw we have, two first SDI signals are reserved for external HDA codecs while 3rd signal is for iDisp. In practise assumption on 0x4 is used throughout the codebase now. I tried to align the code with generic HDA driver, using same names and expressoins, so it's easier to see the similarity.

Reply:
thanks ! I finally get the answer.